### PR TITLE
KC Anchor (Update)

### PR DESCRIPTION
**Pull request by ThatOneGoblin25**
## About the PR
- reworked atmos
- reworked pipes
- reworked wiring (SMES are looped to avoid AME flickering)
- reworked med, no more cryo and chem
- removed medical doctor locker
- removed nanomed
- removed booze & soda dispenser
- removed botany & kitchen
- removed vendors
- replaced blastdoors in bridge with shutters
- replaced chief engineer locker with engineer locker
- replaced T3 gunsafe with empty captain safe
- added two empty gun racks
- added boxing ring
- added disposal units around the ship
- added chaplain EVA and belt
- added janitor closet with disposals
- added television
- added two more arcades
- added cryosleep unit to morgue
- added surgery bed and body scanner (futureproofing:tm:)
- added telecomms for expedition
- decals and minor stuff
- shipyard price lowered

## Why / Balance
Good ship that needs a good update to return to the shipyard.

## Media
anchor_2

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: KC Anchor returns to the expedition shipyard with a rework.

### CODE REVIEW

**Comment by ThatOneGoblin25** [comment]
Hey @Kesiath, letting you know there's some proposed changes to your ship.

**Comment by dvir001** [comment]
image
No medical beds, no medical kits, a single **wall locker** and that's it.

image
They will always spawn looking weird as spawners

image
No

**Comment by ThatOneGoblin25** [comment]
> No medical beds, no medical kits, a single wall locker and that's it.

Not even medical beds? Damn. As for the single wall locker it has the same fill as the medicine locker that's there.

> They will always spawn looking weird as spawners

image
I personally think they look better than they do sideways because you get to see the screen.

> No

Missed that one.

**Comment by ThatOneGoblin25** [comment]
Screenshot 2024-07-16 211750

**Comment by ThatOneGoblin25** [comment]
This is fine.
anchor_2

**Comment by dvir001** [comment]
image

We avoid putting none table machines in tables since it cannot be done normally, need to be moved.

Other than that I don't see anything else that might be an issue from the pictures, leaving this to another maintainer review and merge.

**Comment by ThatOneGoblin25** [comment]
> We avoid putting none table machines in tables since it cannot be done normally, need to be moved.

Alright.

> Fix grid atmos again?

The tiles are like that intentionally, inspired by cluster station.

**Comment by ThatOneGoblin25** [comment]
Screenshot 2024-07-18 233237

**Review by ErhardSteinhauer — CHANGES_REQUESTED** [review]
Shuttle misses cargo palettes which makes it impossible to get rewards from expeditions.
Air scrubber in "sauna" syphons out water vapor.
The material reclaimer setup in custodian room is odd: it throws reclaimed usable materials into space.

**Comment by ThatOneGoblin25** [comment]
> Shuttle misses cargo palettes which makes it impossible to get rewards from expeditions. 

Screenshot 2024-07-22 170504
Ah so that's what there for, gotcha.

Air scrubber in "sauna" syphons out water vapor.

Screenshot 2024-07-22 170217
It's now just its own loop to simplify it. There's a portable scrubber if theres a ammonia buildup ever

The material reclaimer setup in custodian room is odd: it throws reclaimed usable materials into space.

Screenshot 2024-07-22 170445
Should be fine now, can still sort out the junk not reclaimed

**Comment by ThatOneGoblin25** [comment]
anchor3
MapRender of final ver

**Comment by K4rlox** [comment]
image
way too many removals
theres no reason to remove any of these, you are just making it so its not even worth buying this ship anymore
whats the point of buying this ship after you removed all the roleplay value?

previously people had the reason to hire a bartender and a cook for roleplay, if this PR gets merged, they no longer need to

you replaced entirety of kitchen with a useless boxing room that adds nothing expect of reducing ship value and roleplay value


additionally, KC anchor is previously known as the only ship who has a dedicated room for a chaplain, not anymore


these changes feel like it was made to remove all roleplay value and discourage players from ever buying the ship

**Comment by ThatOneGoblin25** [comment]
I was told for any rework merge to be accepted I had to completely remove all vendors, kitchen and botany. 

> additionally, KC anchor is previously known as the only ship who has a dedicated room for a chaplain, not anymore

I didn't actually know this, I'll revisit adding more for chaplain

> you replaced entirety of kitchen with a useless boxing room that adds nothing expect of reducing ship value and roleplay value

Boxing ring felt like a cute rp idea for hard-headed mercs to sort out disputes for reward claiming. I also thought it would be a nice addition since the dedicated boxing ring ship, Knuckleverse was recently removed.

**Review by ErhardSteinhauer — APPROVED** [review]

File: Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-  # description: A large luxury cruiser capable of long ranged travel acrossed the sector, expedition capable.
-  # price: 135020
-  # category: Large
-  # group: Expedition
-  # shuttlePath: /Maps/_NF/Shuttles/Expedition/anchor.yml
+- type: vessel
+  id: Anchor
+  name: KC Anchor
+  description: A large luxury cruiser capable of long ranged travel acrossed the sector, expedition capable.
+  price: 110100
```
Comment (ErhardSteinhauer) [inline Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml:5]: Make sure that the markup is 30%

**Review by ErhardSteinhauer — DISMISSED** [review]

File: Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml (hunk 1 above)
Comment (jimmy12or) [reply Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml:5]: For what? No one is going to buy it at a high price anymore.

Comment (ThatOneGoblin25) [reply Resources/Prototypes/_NF/Shipyard/Expedition/anchor.yml]: Increased price by 7k for 30% markup. Grid sells for slightly less than 90k ingame.

**Review by ErhardSteinhauer — APPROVED** [review]
Looks good to me

**Review by arimah — CHANGES_REQUESTED** [review]
A tiny number of tiny things:

image
This medical locker should probably not be filled.

image
The ship doesn't need _this many_ cargo pallets... You can probably reduce it to 4 or even 2.

image
Random arcade spawner has a chance of spawning a filled space villain arcade game. You can win plushies from it. I'd suggest replacing the spawners with an empty space villain arcade and an NT block game (facing west).

---

Beyond that, I can't see any technical issues with the ship. With so much being removed, unfortunately it feels like a big, dead space that doesn't really offer anything. I'm not sure who would really pick the Anchor, but if it's fine to have a mostly purposeless huge ship, then alright.

**Comment by ThatOneGoblin25** [comment]
> This medical locker should probably not be filled.

Dvir informed me I was allowed to have one filled medicine cabinet on the ship. I think it's fine to have a tiny amount of meds of a expo ship this large.

> The ship doesn't need this many cargo pallets... You can probably reduce it to 4 or even 2.

I'll reduce it to 4 in the corner

> Random arcade spawner has a chance of spawning a filled space villain arcade game. You can win plushies from it. I'd suggest replacing the spawners with an empty space villain arcade and an NT block game (facing west).

Alright.

> Beyond that, I can't see any technical issues with the ship. With so much being removed, unfortunately it feels like a big, dead space that doesn't really offer anything. I'm not sure who would really pick the Anchor, but if it's fine to have a mostly purposeless huge ship, then alright.

Yes unfortunately having to remove botany, kitchen and medical makes this ship quite a bit lackluster, it was a requirement for any approval from Dvir for it to lose it's multirole. However, for its size it's relatively cheap and if any crews want to set up their own space for stuff they have a plethora of options. I considered adding a salvage bay but I didn't really feel like it fit. If you had any suggestions to give it more purpose I'm welcome to suggestions, I just felt like this ship was too iconic to just leave removed.

**Comment by arimah** [comment]
> Dvir informed me I was allowed to have one filled medicine cabinet on the ship. I think it's fine to have a tiny amount of meds of a expo ship this large.

Fair enough, then. If Dvir said you can have meds, you can have meds. :)

**Comment by jimmy12or** [comment]
>Random arcade spawner has a chance of spawning a filled space villain arcade game. You can win plushies from it. I'd suggest replacing the spawners with an empty space villain arcade and an NT block game (facing west).

Just asking so I know, what's wrong with being able to win plushies from an arcade machine?

**Comment by arimah** [comment]
> Just asking so I know, what's wrong with being able to win plushies from an arcade machine?

I must issue a correction: I forgot that these machines on Frontier give you prize tickets that you can cash in at Crazy Casey's Casino and honestly, thinking more about it, it doesn't really matter. It's fine, I'm sorry. The south-facing thing is more troubling. It looks super weird!

**Comment by ThatOneGoblin25** [comment]
Added a Chaplain EVA locker and belt 
Screenshot 2024-08-13 at 4 06 55 PM

Replaced loose AME fuel with locker
Screenshot 2024-08-13 at 4 19 39 PM

Reduced cargo pallets to four, rotated arcade machines and other very minor decal things

**Review by arimah — APPROVED** [review]
Seems to work fine, can't find any obvious oopsies. Good work fixing this ship up!

**Pull request merged**